tests/myapp/tests.py

```diff
@@ -100,16 +97,7 @@ def flush(self):
         with open(os.path.join(os.path.dirname(__file__), 'doctests.txt')) as f:
             with tempfile.NamedTemporaryFile() as temp:
                 text = f.read()
-
-                if PY3:
-                    # unicode literals in the doctests screw up doctest on py3.
-                    # this is pretty icky, but I can't find any other
-                    # workarounds :(
-                    text = re.sub(r"""\bu(["\'])""", r"\1", text)
-                    temp.write(b(text))
-                else:
-                    temp.write(text)
-
+                temp.write(text.encode("latin-1"))
                 temp.flush()
```

Comment:
Is the temporary file even necessary at all now?

Reply:
No, it's not :)